<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>949</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Sokoban</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../graphics/icon.png</normaloff>../graphics/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Уровень: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_level">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="restart_level_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Заново</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../graphics/restart.png</normaloff>../graphics/restart.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_back">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Назад</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../graphics/back.png</normaloff>../graphics/back.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Сделано ходов:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_move">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>700</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Уровень</string>
    </property>
    <addaction name="select_level"/>
    <addaction name="restart_level"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="about1"/>
    <addaction name="help_act"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Импорт</string>
    </property>
    <addaction name="import_txt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="select_level">
   <property name="icon">
    <iconset>
     <normaloff>../graphics/choice.png</normaloff>../graphics/choice.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать..</string>
   </property>
  </action>
  <action name="restart_level">
   <property name="icon">
    <iconset>
     <normaloff>../graphics/restart.png</normaloff>../graphics/restart.png</iconset>
   </property>
   <property name="text">
    <string>Заново</string>
   </property>
  </action>
  <action name="about1">
   <property name="icon">
    <iconset>
     <normaloff>../graphics/about.png</normaloff>../graphics/about.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="help_act">
   <property name="icon">
    <iconset>
     <normaloff>../graphics/help.png</normaloff>../graphics/help.png</iconset>
   </property>
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="import_txt">
   <property name="icon">
    <iconset>
     <normaloff>../graphics/import_db.png</normaloff>../graphics/import_db.png</iconset>
   </property>
   <property name="text">
    <string>Импорт из текстового файла</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
